django/shortcuts.py

```diff
@@ -188,3 +188,7 @@ def resolve_url(to, *args, **kwargs):
 
     # Finally, fall back and assume it's a URL
     return to
+
+
+def make_toast():
+    return "toast"
```

Comment:
I don't think we need this here!

Reply:
"We certainly do not. I will remove all references to this function!"